docs/organizations/managing-security-and-risk.md

```diff
@@ -221,12 +221,9 @@ To revoke the Security Manager role:
 
 1.  Click the **Revoke role icon** to the right of the user's name and confirm.
 
-<!--
-TODO Update this section once the behavior is consistent. See https://codacy.slack.com/archives/C050XTVETSQ/p1687167901474059
-
 ## Data retention
 
-Codacy retains open items indefinitely and closed items for one year.
+Codacy retains all items indefinitely.
```

Comment:
This sentence seems to contradict the details below for Codacy/Jira integrations... 🤔 

(@nicklem I know this pull request is already merged. Leaving this comment for your evaluation.)

Reply:
Ignore my comment, you solved it on #1820 ! 😅